server/fleet/policies.go

```diff
@@ -184,6 +184,15 @@ type PolicyCalendarData struct {
 	Name string `db:"name" json:"name"`
 }
 
+// PolicyLite is a stripped down version of the policy.
+type PolicyLite struct {
+	ID uint `db:"id" json:"id"`
```

Comment:
re json: is this type exposed to the API?

Reply:
No. I'll remove `json`